symfony/http-kernel/CVE-2015-4050.yaml

```diff
@@ -14,4 +14,7 @@ branches:
     2.6.x:
         time:     2015-05-26 23:55:51
         versions: [>=2.6.0,<2.6.8]
+    2.7.x:
+        time:     2015-05-26 23:55:51
+        versions: [>=2.7.0-dev,<2.7.0]
```

Comment:
I see, thanks for clarification. I did not know about this syntax, and I did not find it in the documentation, at least [here](https://getcomposer.org/doc/01-basic-usage.md#package-versions) it is not covered.

There is still a problem with the contraint. This matches:
http://semver.mwl.be/#?package=symfony%2Fsymfony&version=%3E%3D2.7.0-dev&minimum-stability=dev
but this does not match anymore:
http://semver.mwl.be/#?package=symfony%2Fsymfony&version=%3E%3D2.7.0-dev,%3C2.7.0&minimum-stability=dev